lib/board.rb

```diff
@@ -0,0 +1,23 @@
+# require 'game'
+class Board
+  def initialize
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+  end
+
+  def board_string
+
+    expected_output  = " #{@board[0]} | #{@board[1]} | #{@board[2]} \n"
+    expected_output += "-----------\n"
+    expected_output += " #{@board[3]} | #{@board[4]} | #{@board[5]} \n"
+    expected_output += "-----------\n"
+    expected_output +  " #{@board[6]} | #{@board[7]} | #{@board[8]} \n"
+  end
+
+  def position_taken?(index)
+    @board[index] != ' '
+  end
+
+  def mark_board(index, char = 'X')
```

Comment:
1. Why did you name the second parameter `char`? Can you think of a name that is more related to the domain (you're marking a space on a tic tac toe board - what might you call the thing that you're marking it with)? For this kind of program it doesn't really matter, but it's often helpful to avoid naming variables/parameters things like `string` or `char`, and instead give them descriptive names that relate to the purpose of the function from a user's perspective. 
2. When you call this function from another object, it will look probably look like `board.mark_board(2, 'O')` - what do you think about changing the name to just be `mark`?

Reply:
@Middlebrooks314 You resolved this without answering my questions 🙁

Reply:
I used char because the Trello card said it should allow any character, I was going to probably refactor that once a story had more strict requirements for X & O.